### DISCUSSION TITLE
Left-align bullet points / ordered lists

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [marp-team/marp#459] [Q&A] [answered]
How can I left-align bullet points / ordered lists?

I've tried the following which left aligns titles and paragraphs"
```
section {
  text-align: left;
  font-family: arial;
  justify-content: start;
}
```

**Comment by yhatt** [comment]
Every list items should be already left aligned through `text-align: left` by default. Which Marp theme are you using?

**Comment by SoumayaMauthoorMOJ** [comment]
I'm using `uncover`:

![image](https://github.com/marp-team/marp/assets/95432492/e408a833-895f-4c1e-955e-b039d43bc2a4)

**Comment by SoumayaMauthoorMOJ** [comment]
Bullet lists with longer text are left-aligned if that makes a difference

**Comment by SoumayaMauthoorMOJ** [comment]
I get the same issue even if I remove all styling:

![image](https://github.com/marp-team/marp/assets/95432492/8611c0d4-0442-43e2-afd1-8a5fca1f9884)

**Comment by yhatt** [comment, marked as answer]
`uncover` theme is using a different trick `margin: 0 auto` than `text-align` and flex related CSS styles for centering lists.
https://github.com/marp-team/marp-core/blob/e05e5d5faad74b4b359377cc46303ba388ecd6ae/themes/uncover.scss#L168

To reset centering, set horizontal margin for the bullet list `<ul>` and ordered list `<ol>` to `0`.

```css
ul, ol {
  margin-left: 0;
  margin-right: 0;
}
```

ref: https://www.w3.org/Style/Examples/007/center.en.html

**Comment by SoumayaMauthoorMOJ** [comment]
Thank you!
